Source/WebCore/dom/DocumentMarkerController.cpp

```diff
@@ -96,6 +96,13 @@ void DocumentMarkerController::addDraggedContentMarker(const SimpleRange& range)
         addMarker(textPiece.node, { DocumentMarker::Type::DraggedContent, textPiece.range, RefPtr<Node> { textPiece.node.ptr() } });
 }
 
+void DocumentMarkerController::addTransparentContentMarker(const SimpleRange& range, WTF::UUID uuid)
+{
+    // FIXME: Since the marker is already stored in a map keyed by node, we can probably change things around so we don't have to also store the node in the marker.
+    for (auto& textPiece : collectTextRanges(range))
+        addMarker(textPiece.node, { DocumentMarker::Type::TransparentContent, textPiece.range,  DocumentMarker::TransparentContentData { RefPtr<Node> { textPiece.node.ptr() } , uuid } });
```

Comment:
Nit - stray whitespace. I also don't think you need to explicit `RefPtr<Node> { … }`

```suggestion
        addMarker(textPiece.node, { DocumentMarker::Type::TransparentContent, textPiece.range, DocumentMarker::TransparentContentData { { textPiece.node.ptr() }, uuid } });
```